kinoml/smth.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+
+
+def my_function(param1: int, param2: Optional[str] = None) -> str:
+    """
+    A short description of this function.
+
+    A longer description of this function.
+    You can use more lines.
+
+        This is code block,
+        as usual.
+
+    ```python
+    s = "This is a Python code block :)"
+    ```
+
+    Arguments:
```

Comment:
@jchodera, are we sure we want to go and use this syntax for our library? I wouldn't mind at all, but we have to make the decision now before I start writing too many docstrings.